README.md

```diff
@@ -22,21 +22,21 @@ in Internet Explorer on Windows 7 for instance (because of the wrong MIME type).
 
 The plugin can also be loaded as AMD or CommonJS module.
 
-## Usage
+## Basic Usage
 
-Create session cookie:
+Create a session cookie, valid to the current page:
```

Comment:
My theory is that "on" is when it is above, "in" when it's inside, "to" when refers to. I mean, that's just from immersive experience I am not used to check online references as much as for technical issues xD